cmd/juju/application/consume.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/api/application"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/core/crossmodel"
+)
+
+var usageConsumeSummary = `
+Add a remote application to the model.`[1:]
+
+var usageConsumeDetails = `
+Adds a remote application to the model. Relations can be created later using "juju relate".
+
+The remote application can be identified in two ways:
+    [<model owner>/]<model name>.<application name>
+        for an application in another model in this controller (if owner isn't specified it's assumed to be the logged-in user)
+or
+    <remote endpoint url>
+        for remote applications that have been shared using the offer command
+
+Examples:
+    $ juju consume othermodel.mysql
+
+    $ juju consume local:/u/fred/db2
+
+See also:
+    add-relation
+    offer`[1:]
+
+// NewConsumeCommand returns a command to add remote applications to
+// the model.
+func NewConsumeCommand() cmd.Command {
+	return modelcmd.Wrap(&consumeCommand{})
+}
+
+// consumeCommand adds remote applications to the model without
+// relating them to other applications.
+type consumeCommand struct {
+	modelcmd.ModelCommandBase
+	api               serviceConsumeAPI
```

Comment:
Oh yeah. I actually didn't realise that was what that service prefix referred to - the service/application change was just when I started, so it's not as visceral for me.

Fixed.